### DISCUSSION TITLE
A package can contain 0 or more applications

### DISCUSSION THREAD

**Discussion by jaigak** [microsoft/WindowsAppSDK#5464] [Ideas]
Some apps like Mail & Calendar share the same package but they run as different apps. Although this is possible for everyone, Microsoft doesn't provide proper tooling which makes it difficult to build multi-app packages. Since the package is tied to the project in Visual Studio you need to somehow copy the output files from another project and then you need to modify the applications tag to include the another application and then deploy. Windows Application Packaging Project only supports one UWP application and one full trust.

**Comment by RDMacLachlan** [comment]
What kind of tooling should be provided to Software Developers to support this scenario.
